spec/presenters/child_count_display_fetcher_spec.rb

```diff
@@ -24,7 +24,7 @@
     end
 
     describe "with no members" do
-      let(:item) { create(:public_work) }
+      let(:item) { build(:public_work, members: []) }
```

Comment:
In the branch you abandoned this was: 

    let(:item) { build(:public_work, members: []).tap {|w| w.save!(validate: false) } }

I thought that was necessary to actually get a public_work with no members, since otherwise... aha, this branch eliminates the validation! Right. OK.

Reply:
Yeah - no longer needed...